src/ObjectUtils/include/ObjectUtils/ElfFile.h

```diff
@@ -53,7 +53,8 @@ class ElfFile : public ObjectFile {
   //
   // This method returns load bias for the elf-file if program headers are
   // available. This should be the case for all loadable elf-files.
-  [[nodiscard]] virtual ErrorMessageOr<uint64_t> GetLoadBias() const = 0;
+  [[nodiscard]] virtual uint64_t GetLoadBias() const = 0;
+  [[nodiscard]] virtual uint64_t GetExecutableSegmentOffset() const = 0;
```

Comment:
Is this a function you need for the absolute address changes you are working on? (Just wondering, as this is not yet used anywhere other than tests it seems.)

Reply:
Yes, it is going to be used in the next PR; the initial change was too big - so I split it to 2.